File: QE-Selenium-Java/Activity13.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

import java.util.List;

public class Activity13 {
    public static void main(String[] args) {

        FirefoxDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/tables");
        System.out.println(driver.getTitle());

        List<WebElement> rows = driver.findElements(By.xpath("//table/tbody/tr"));
        System.out.println(rows.size());

        List<WebElement> cols = driver.findElements(By.xpath("//table/thead/tr/th"));
        System.out.println(cols.size());

        List<WebElement> third_row = driver.findElements(By.xpath("//table/tbody/tr[3]/td"));
        for (WebElement cell : third_row) {
            System.out.println(cell.getText());
        }

        WebElement cell_value = driver.findElement(By.xpath("//table/tbody/tr[2]/td[2]"));
        System.out.println(cell_value.getText());

        driver.quit();
    }
}

File: QE-Selenium-Python/Activity13.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/tables") 
print(driver.title)
rows=driver.find_elements(By.XPATH,"//table/tbody/tr")
print(len(rows))
cols=driver.find_elements(By.XPATH,"//table/thead/tr/th")
print(len(cols))
third_row=driver.find_elements(By.XPATH,"//table/tbody/tr[3]/td")
for cell in third_row:
    print(cell.text)
cell_value=driver.find_element(By.XPATH,"//table/tbody/tr[2]/td[2]")
print(cell_value.text)
driver.quit()
